runtime.md

```diff
@@ -122,8 +137,3 @@ This specification does not mandate the name of this JSON file.
 See the specification of the `config.json` file for the definition of these fields.
 The stopping, or exiting, of these secondary process MUST have no effect on the state of the container.
 In other words, a container (and its PID 1 process) MUST NOT be stopped due to the exiting of a secondary process.
-
-## Hooks
-
-Many of the operations specified in this specification have "hooks" that allow for additional actions to be taken before or after each operation.
```

Comment:
Let me try to summarize the plan for supporting post-stop hooks:
1. Create a container `A`
2. Create another container `B` that will be reuse the namespaces (excepting pid?) from `A`
3. Start, then eventually stop/delete `A`.
4. Since `B` is expected to be still active, once can exec into `B` and run post stop hooks.

Some thoughts & question on the proposed approach.
1. What parts of `A`'s environment is sharable with `B`?
2. When the init pid in `A` dies what parts of the sandbox is allowed to be cleaned up? For example, can `B` use `A`s cgroups and will cgroups be preserved?
3. Can `B` be created without a pid namespace? If we rely on pid namespace, there is a chance that the init process we launch as part of `Create` might die..

Overall, although the proposed approach sounds somewhat plausible, it is very cumbersome. The API and hence the clients that use the API, will be much cleaner if we were able to preserve (parts) the sanbox even after the init process dies.